src/components/homepage.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { BiRightArrowCircle } from 'react-icons/bi';
+import { date } from '../redux/slices/covidSlice';
+import { filterByCountry } from '../redux/covid/covid';
+import style from './homepage.module.css';
+
+const HomePage = () => {
+  const [select, setSelect] = useState('Afghanistan');
+  const dispatch = useDispatch();
+  const state = useSelector((state) => state.covid);
+  const covidTotal = state.data.total;
+  let countries;
+  let countriesKeys;
+  if (state.data.dates) {
+    countries = state.data.dates[date].countries;
+    countriesKeys = Object.entries(countries);
+  }
+  const handleCategory = (event) => {
+    setSelect(event.target.value);
+  };
+  return (
+    <div className={style.div}>
+      { state.loading
+      && (
+      <div className={style.loadingDiv}>
+        <h2 data-testid="loading" className={style.loading}>Loading, please wait</h2>
+        <div className="loadingio-spinner-reload-zln0owzosys">
+          <div className="ldio-cf5nk0nv464">
+            <div>
+              <div />
+              <div />
+              <div />
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] Kindly have this rendered in another component and import it into the home component. This would allow you to have a reusable component that does a specific task.